getting_started/access_your_services.adoc

```diff
@@ -59,7 +59,46 @@ https://logs.<cluster-id>.openshift.com
 ----
 https://api.<cluster-id>.openshift.com/oauth2callback/<IdP name>
 ----
+endif::[]
+
+ifdef::openshift-arho[]
+[[access-your-cluster-overview]]
+== Access your cluster
+
+// TODO Jim Minter is to provide updated URL info for ARHO
+
+Once your {product-title} cluster is configured and ready to use, you can access
```

Comment:
Need to say that all of the URLs include a cluster ID component (chosen by the service) and a region component (where the user provisioned the cluster)